store/localcachelayer/channel_layer.go

```diff
@@ -132,3 +149,32 @@ func (s LocalCacheChannelStore) GetPinnedPostCount(channelId string, allowFromCa
 
 	return count, nil
 }
+
+func (s LocalCacheChannelStore) Get(id string, allowFromCache bool) (*model.Channel, *model.AppError) {
+	return s.get(id, false, allowFromCache)
+}
+
+func (s LocalCacheChannelStore) get(id string, master bool, allowFromCache bool) (*model.Channel, *model.AppError) {
+
+	if allowFromCache {
+		if cacheItem := s.rootStore.doStandardReadCache(s.rootStore.channelByIdCache, id); cacheItem != nil {
+			if s.rootStore.metrics != nil {
+				s.rootStore.metrics.IncrementMemCacheHitCounter("Channel")
+			}
```

Comment:
The hit counter is already incremented by calling `rootStore.doStandardReadCache()`. We can safely remove this block.